src/BitStringAddresses/fermifs.jl

```diff
@@ -131,6 +131,54 @@ Base.:(==)(a::FermiFS, b::FermiFS) = a.bs == b.bs
 num_occupied_modes(::FermiFS{N}) where {N} = N
 occupied_modes(a::FermiFS{N,<:Any,S}) where {N,S} = FermiOccupiedModes{N,S}(a.bs)
 
+num_unoccupied_modes(::FermiFS{N,M}) where {N,M} = M - N
+function unoccupied_modes(a::FermiFS{N,M,S}) where {N,M,S}
+    s = ~a.bs
+    FermiUnoccupiedModes{M - N,typeof(s)}(s)
+end
+
+"""
+    unoccupied_mode_map(addr::FermiFS) <: AbstractVector
+    
+Get a map of unoccupied modes in [`FermiFS`](@ref) address as an `AbstractVector`
+of indices compatible with [`excitation`](@ref).
+
+`unoccupied_mode_map(addr)[i]` contains the index for the `i`-th unoccupied mode.
+This is useful because unoccupied modes is required in some cases.
+`unoccupied_mode_map(addr)` is an eager version of the iterator returned by
+[`unoccupied_modes`](@ref). It is similar to 
+[`onr`](@ref) but contains more information.
+
```

Comment:
```suggestion

Note that this function is only implemented for addresses of type [`FermiFS`](@ref).

```
We don't really need this function for bosonic addresses, but I'd add this comment to the docstring just to make it even more explicit.